api/client/proto/authservice.proto

```diff
@@ -1099,6 +1102,11 @@ service AuthService {
     // DeleteToken deletes an existing token in a backend described by the given request.
     rpc DeleteToken(types.ResourceRequest) returns (google.protobuf.Empty);
 
+    // GetClusterAuditConfig gets cluster audit configuration.
+    rpc GetClusterAuditConfig(google.protobuf.Empty) returns (types.ClusterAuditConfigV2);
+    // SetClusterAuditConfig sets cluster audit configuration.
+    rpc SetClusterAuditConfig(types.ClusterAuditConfigV2) returns (google.protobuf.Empty);
```

Comment:
`ResetClusterConfig` will be added in the next PR that adds `tctl` bindings, right?

Reply:
`ClusterAuditConfig` isn't supposed to be dynamically configurable (at least for now) so the answer is no.